lib/bulk_data_test_kit/v1.0.1/bulk_data_status_check_test.rb

```diff
@@ -22,6 +22,24 @@ class BulkDataStatusCheckTest < Inferno::Test
       DESCRIPTION
       # link 'http://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#bulk-data-status-request'
 
+      verifies_requirements 'hl7.fhir.uv.bulkdata_2.0.0@123',
+                            'hl7.fhir.uv.bulkdata_2.0.0@125',
```

Comment:
The retry-after header [is used](https://github.com/inferno-framework/bulk-data-test-kit/blob/37c3a4b447b1a69d9b8fb08a0a3b81460ef35629/lib/bulk_data_test_kit/export_operation_tests.rb#L112) but not tested for. It was unclear to me whether to add the reference to 125 or not since it looks like the function would have errors if it were not present. Or perhaps the thing to do here is to remove the reference to 125 and then update the test kit to test for the existence of the header, then add it back at a later date?